extensions/default/src/DicomWebDataSource/retrieveStudyMetadata.js

```diff
@@ -38,9 +39,11 @@ export function retrieveStudyMetadata(
     throw new Error(`${moduleName}: Required 'StudyInstanceUID' parameter not provided.`);
   }
 
+  const promiseId = (dicomWebConfig.name || '') + StudyInstanceUID;
```

Comment:
In general making it look URI ish is nice => how about `${dicomWebConfig.name}:${StudyInstanceUID}`

Reply:
Updated, thanks!